cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
  <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.5.1
+
+_Released 02/10/2023 (PENDING)_
+
+**Dependency Updates:**
+
+- Upgraded [`simple-git`](https://github.com/steveukx/git-js) from `3.15.0` to `3.16.0` to address this [security vulnerability](https://github.com/advisories/GHSA-9p95-fxvg-qgq2) where Remote Code Execution (RCE) via the clone(), pull(), push() and listRemote() methods due to improper input sanitization was possible. Fixed in [#25603](https://github.com/cypress-io/cypress/pull/25603).
```

Comment:
```suggestion
- Upgraded [`simple-git`](https://github.com/steveukx/git-js) from `3.15.0` to `3.16.0` to address this [security vulnerability](https://github.com/advisories/GHSA-9p95-fxvg-qgq2) where Remote Code Execution (RCE) via the clone(), pull(), push() and listRemote() methods due to improper input sanitization was possible. Addressed in [#25603](https://github.com/cypress-io/cypress/pull/25603).
```

Reply:
Thought I had this right the first time: https://github.com/cypress-io/cypress/pull/25603/commits/e16fb5dca9bc228c514db9502d31ea8c28158c1b, but then I just kept messing it up 😅

Reply:
@ZachJW34 the PR title started with `fix:` which is why it complained I think

Reply:
though I want to double check this guys cuz this made it in wrong: https://github.com/cypress-io/cypress/pull/25574

Reply:
Makes sense! Thanks for fixing that up for me.

Can we get renovate to default to these titles?